flutter/cpp/utils.cc

```diff
@@ -105,6 +106,11 @@ mlperf_backend_configuration_t CppToCSettings(const SettingList &settings) {
   strcpy(accelerator, settings.benchmark_setting().accelerator().c_str());
   c_settings.accelerator = accelerator;
   c_settings.batch_size = settings.benchmark_setting().batch_size();
+  char *accelerator_desc =
+      new char[settings.benchmark_setting().accelerator_desc().length() + 1];
```

Comment:
Please check this https://sonarcloud.io/project/security_hotspots?id=mobile_app_open&pullRequest=567&hotspots=AYPB2Rc8EkE0XNv3oXkz